jekyll/_cci2/migrating-from-1-2.md

```diff
@@ -3,332 +3,133 @@ layout: classic-docs
-    docker:
-
-      # In 2.0, you can use multiple different images and group them as a 'Pod'.
-      # Because Docker containers normally don't include DBs, you need to bring
-      # a build image that serves as DB container in your build.
-      - image: primary-build-image
-      - image: db-image-1
-      - image: db-image-2
-
-    steps:
-      - checkout
-
-      # We're still building our new inference system for 2.0, but until then, there’s nothing to override.
-      # For now, you'll need to manually install your project’s dependencies.
-      # The current timeout for no output is hardcoded to 600 seconds
-      - run:
-          name: Install Dependencies
-          command: dependency-install-command
-
-      # The DB is not automatically created in 2.0, so we manually set up a test DB
-      - run:
-          name: Create Test DB
-          command: setup-db-command
-
-      # Again, no inference in 2.0 yet, so nothing to override here, either.
-      # Just use any test commands.
+```
       - run:
-          name: Run Tests
-          command: test-command
+          name: <name>
+          command: <installed-dependency>
 ```
 
-## Caching
-
-### Cache Directories
-
-**1.0**
-
-```YAML
-dependencies:
-  override:
-    - bundle install --path vendor/bundle:
-        timeout: 180
+- Search and replace the `cache_directories:` key:
 
+```
   cache_directories:
     - "vendor/bundle"
 ```
 
-**2.0**
+With the following, nested under `steps:` and customizing for your application as appropriate:
 
-{% raw %}
-```YAML
-version: 2
-jobs:
-  build:
-    working_directory: /root/my-project
-    docker:
-      - image: ruby:2.3
-
-    steps:
-      - checkout
-      - run: bundle install --path vendor/bundle
-
-     # Caching is fully customizable in 2.0
+```
      - save_cache:
-        key: dependency-cache-{{ checksum "Gemfile.lock" }}
+        key: dependency-cache
         paths:
           - vendor/bundle
 ```
-{% endraw %}
-
-Please note that you need to have a restore cache step by yourself in 2.0. There are also lots of cache prefix options available.
-
-Read about them in [Configuration Reference]({{ site.baseurl }}/2.0/configuration-reference/#save_cache).
 
-## Deployment
+- Add a `restore_cache:` key nested under `steps:`.
 
-**1.0**
+- Search and replace `deployment:` with the following, nested under `steps`:
 
-```YAML
-deployment:
-  staging:
-    branch: master
-    heroku:
-      appname: foo-bar-123
+```
+     - deploy:
 ```
 
-**2.0**
-
-Currently, 2.0 doesn't support automatic deployment via integrations (like the above Heroku example).
-
-You can write your own manual `deploy` steps as shown in the [Configuration Reference]({{ site.baseurl }}/2.0/configuration-reference/#deploy).
+**Note:** Currently, 2.0 Beta does not include integration support. To write your own manual `deploy` steps, refer to the [Configuration Reference]({{ site.baseurl }}/2.0/configuration-reference/#deploy).
```

Comment:
per Nathan, this should be referred to as 'continuous deployment' instead of integration support.